domains/gb.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "your-github-username",
+        "email": "your-email-address@example.com"
+    },
+    "record": {
+        "CNAME": "https://my-portfolio-swart-eight.vercel.app/"
```

CNAME can't contain protocol or slashes
```suggestion
        "CNAME": "my-portfolio-swart-eight.vercel.app"
```